src/test/a11y/aggregateAccessibilityResults.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const path = require('path');
+const htmlFilesDir = 'functional-output/accessibility';
+const outputFile = 'functional-output/accessibility/accessibility_results.html';
+
+
+function aggregateAccessibilityResults() {
```

Comment:
is this based on an example? Where's it from?

Reply:
It's based on nodejs file-system library.